.ddev/config.yaml

```diff
@@ -0,0 +1,160 @@
+APIVersion: v1.12.0-alpha2
+name: TYPO3-Console
+type: typo3
```

Comment:
do we need this to be type TYPO3 or can it be just PHP?

Reply:
This will only ensure that the AdditionalConfiguration.php with the default values for db, mailhog, etc see https://github.com/drud/ddev/blob/master/pkg/ddevapp/typo3.go#L17

If you do not like this file being generated we can switch to php...